plugin/openassetio_manager_bal/BasicAssetLibraryInterface.py

```diff
@@ -147,27 +155,35 @@ def isEntityReferenceString(self, someString, hostSession):
         return someString.startswith(self.__entity_refrence_prefix())
 
     @simulated_delay
-    def entityExists(self, entityRefs, _context, _hostSession, successCallback, errorCallback):
-        results = []
+    def entityExists(self, entityRefs, context, _hostSession, successCallback, errorCallback):
+        if not self.__validate_access(
+            "entityExists",
+            (Context.Access.kRead,),
+            context,
+            entityRefs,
+            errorCallback,
+        ):
+            return
```

Comment:
As a side note - this is a good example of why the new approach will clarify things. Access isn't meant to be sensitive to access, and the only way you'd know is reading the docs...

Reply:
[fix] Capturing offline conversation remove for now as well remove it shortly anyway

Reply:
Removed in 5feb74be

Reply:
As discussed, resolving on behalf. Still blocked on https://github.com/OpenAssetIO/OpenAssetIO/issues/1016 but will rebase in the meantime.